dist/adguard.json

```
@@ -96,6 +96,12 @@
             "url": "https://www.markmonitor.com/",
             "companyId": "markmonitor"
         },
+        "outlook": {
+            "name": "Microsoft Outlook",
+            "categoryId": 10,
```

```suggestion
            "categoryId": 13,
```

Category should be `13` -- Email